cfdtools/data.py

```diff
@@ -124,34 +122,34 @@ def dataSet_spectrogram(self, datafilter=None, window=None):
         datalist.remove('time')
         dtavg, dtdev, ntot = self.dtstats()  # check 'timeevol'
         # print(dtavg, dtdev)
-        if dtdev / dtavg > 1.e-6:
-            api.io.warning(
-                f"dt standard deviation is significant: {100*dtdev / dtavg:.4f}%"
-            )
+        if dtdev / dtavg > 1.0e-6:
+            api.io.warning(f"dt standard deviation is significant: {100*dtdev / dtavg:.4f}%")
```

Comment:
100 * dtdev

Reply:
to make it a percentage

Reply:
Side effect of Black code formatter on the dev. Not my part but question still pending for next commits?

Reply:
C'était pour les espaces autour de '`*`' comme en ligne 132 ci-dessous (formatage général de ce type dans ce fichier, avec `black code` ?)

Reply:
Yes.